### ISSUE TITLE
最新好像很久没更新了,是出什么问题了吗,我很担心啊

### ISSUE THREAD

**Issue by zhenhappy** [XX-net/XX-Net#4149]

**Comment by jzp820927** [comment]
X大退居了，使用基本上不更新了。除非有重大问题。

**Comment by wooowooo** [comment]
怎么会这样,这两个月以来突然不活跃了

**Comment by jzp820927** [comment]
只要能用就行了吧，如果突然有一天不能用了，马上就会活跃了，放心吧

**Comment by wooowooo** [comment]
谁会是这个项目的继承项目?

**Comment by jzp820927** [comment]
goproxy，不过不带扫描

**Comment by yaoguaishou** [comment]
@jzp820927 
请问:goproxy的linux已编译包里面没有proxy.ini也没有google ip.txt，那么这些东西存储在哪里？

**Comment by jzp820927** [comment]
goproxy 是 go 语言开发的，配置方法和 goagent 完全不同，具体情况，看下面3个链接
简易教程 https://github.com/phuslu/goproxy/blob/wiki/SimpleGuide.md
配置介绍 https://github.com/phuslu/goproxy/blob/wiki/ConfigIntroduce.md
常见问题 https://github.com/phuslu/goproxy/blob/wiki/FAQ.md

**Comment by wooowooo** [comment]
这个goproxy的作者与之前被要求删除的goagent作者是同一人吗?

**Comment by phuslu** [comment]
不是。 goproxy 只是 goagent 的 go语言实现版。goagent 已死， goproxy 也并不能满足期望，如同 xx-net 一样好用。请勿赋予它太多意义。

比如 goproxy 也没有 xx-net 最大的优势(扫描 IP），称它是 xx-net 的继承项目太过抬举。

**Comment by yjd** [comment]
现在xx-net丢服务器上。长期IP延迟	139。
主用ssr备用xx-net

**Comment by abchb99** [comment]
我试过用大半年以前的xx-net版本，都可以正常使用。

**Comment by g8up** [comment]
我觉得把功能拆分未必不是好事，某一工具专门提供 proxy 服务，某工具专门提供扫描。
用户虽然使用成本有增加，但这些工具集不至于因某一高危功能而`一损俱损`。

**Comment by yeahwu** [comment]
@phuslu 社长过谦了，感谢社长一直以来对goproxy的维护。
@zhenhappy
xx-net改自goagent，可以说xx-net、goagent、goproxy都是一家。功能上大相迳庭，goproxy由于社长维护积极，效率突出。
g8up <notifications@github.com>于2016年9月6日 周二17:55写道：

**Comment by theratlesnake** [comment]
@phuslu 写个goproxy的安装教程吗？
